common/realsense-ui-advanced-mode.h

```diff
@@ -203,7 +203,7 @@ struct advanced_mode_control
 };
 
 inline void draw_advanced_mode_controls(rs400::advanced_mode& advanced, 
-    advanced_mode_control& amc, bool& get_curr_advanced_controls, bool& was_set, std::string& error_message)
+    advanced_mode_control& amc, bool& get_curr_advanced_controls, bool& was_set, std::string& error_message, bool mipi_device=false)
```

Comment:
I think it is better to pass device name or connection type and let this function implement the logic of checking it rather then pass a flag.